### ISSUE TITLE
Only watch .go file change

### ISSUE THREAD

**Issue by marcosteam** [cosmtrek/air#151]
I'm using [gofiber/fiber](https://github.com/gofiber/fiber) to write Web Application and Air to hot reload.
But when I'm using pug to my template Engine, Air won't do anything when I change my template.

Here's my `.air.toml` config:

```
# Config file for [Air](https://github.com/cosmtrek/air) in TOML format

# Working directory
# . or absolute path, please note that the directories following must be under root.
root = "."
tmp_dir = "tmp"

[build]
# Just plain old shell command. You could use `make` as well.
cmd = "go build -o ./application ./main.go"
# Binary file yields from `cmd`.
bin = "application"
# Customize binary.
full_bin = "application"
# Watch these filename extensions.
include_ext = ["go", "pug"]
# Ignore these filename extensions or directories.
exclude_dir = ["tmp", "vendor", "node_modules"]
# Watch these directories if you specified.
include_dir = []
# Exclude files.
exclude_file = []
# This log file places in your tmp_dir.
log = "air.log"
# It's not necessary to trigger build each time file changes if it's too frequent.
delay = 1000 # ms
# Stop running old binary when build errors occur.
stop_on_error = true
# Send Interrupt signal before killing process (windows does not support this feature)
send_interrupt = false
# Delay after sending Interrupt signal
kill_delay = 500 # ms

[log]
# Show log time
time = false

[color]
# Customize each part's color. If no color found, use the raw app log.
main = "magenta"
watcher = "cyan"
build = "yellow"
runner = "green"

[misc]
# Delete tmp directory on exit
clean_on_exit = true
```

**Comment by HeCorr** [comment]
Hello! Can you please run air with the `-d` flag and show the output without reloading?

It should say something like `!exclude (...)` a few times.

Also, how's your project structured? Where are the ".pug" templates stored at?

**Comment by marcosteam** [comment]
> Hello! Can you please run air with the `-d` flag and show the output without reloading?
> 
> It should say something like `!exclude (...)` a few times.
> 
> Also, how's your project structured? Where are the ".pug" templates stored at?

Sure.

`air -d`：
```
  __    _   ___  
 / /\  | | | |_) 
/_/--\ |_| |_| \_ v1.12.1 // live reload for Go apps, with Go1.14.0

[debug] mode
CWD: /home/marcosteam/sspanel-go
watching .
watching database
watching middleware
watching middleware/auth
watching middleware/auth/login
watching middleware/auth/register
watching static
watching static/css
watching static/html
watching static/images
watching static/js
watching template
watching template/auth
!exclude tmp
building...


//When pug template file changing:
event: "/home/marcosteam/sspanel-go/template/auth/header.pug": WRITE
event: "/home/marcosteam/sspanel-go/template/auth/header.pug": WRITE

//When go file changing:
event: "/home/marcosteam/sspanel-go/main.go": WRITE
main.go has changed
event: "/home/marcosteam/sspanel-go/main.go": WRITE
main.go has changed
flushing events
main.go has changed
building...
trying to kill cmd [/bin/sh -c /home/marcosteam/sspanel-go/tmp/main]
cmd killed, pid: 2424
running...
```

And my file structure:
```
├── database
├── go.mod
├── go.sum
├── LICENSE
├── main.go
├── middleware
│   └── auth
│       ├── login
│       │   └── login.go
│       └── register
│           └── register.go
├── README.md
├── static
│   ├── css
│   ├── html
│   ├── images
│   └── js
├── template
│   └── auth
│       ├── footer.pug
│       ├── header.pug
│       ├── loginPage.pug
│       └── registerPage.pug
└── tmp
    ├── build-errors.log
    └── main
```

**Comment by HeCorr** [comment]
Interesting.. It really should be working..

Btw, I noticed you are running an older version (see #133). Could you please install the latest version and try again?

Here's the command:  `go install github.com/cosmtrek/air@v1.15.1`

One last thing: I don't think it's necessary to watch the `static` folder, unless the contents are loaded only once on startup.

**Comment by marcosteam** [comment]
> Interesting.. It really should be working..
> 
> Btw, I noticed you are running an older version (see #133). Could you please install the latest version and try again?
> 
> Here's the command: `go install github.com/cosmtrek/air@v1.15.1`
> 
> One last thing: I don't think it's necessary to watch the `static` folder, unless the contents are loaded only once on startup.

It works! Thanks a lot!
I think we should update the documents..

**Comment by HeCorr** [comment]
Awesome! I'm glad it worked  :)
